app/Http/Controllers/AppealController.php

```diff
@@ -112,15 +113,47 @@ public function appeallist()
                 });
         }
 
-        $hiddenStatuses = $isDeveloper
-            ? [Appeal::STATUS_ACCEPT, Appeal::STATUS_DECLINE, Appeal::STATUS_EXPIRE, Appeal::STATUS_INVALID]
-            : [Appeal::STATUS_ACCEPT, Appeal::STATUS_DECLINE, Appeal::STATUS_EXPIRE, Appeal::STATUS_VERIFY, Appeal::STATUS_NOTFOUND, Appeal::STATUS_INVALID];
+        $appealtypes = ['assigned'=>'Assigned to me','unassigned'=>'All unreserved open appeals','reserved'=>'Open reserved appeals'];
+        if($isDeveloper) { $appealtypes['developer']='Developer access appeals'; }
 
-        $appeals = Appeal::whereIn('wiki', $wikis)
-            ->whereNotIn('status', $hiddenStatuses)
+        $developerStatuses = [Appeal::STATUS_VERIFY, Appeal::STATUS_NOTFOUND];
+        $basicStatuses = [Appeal::STATUS_ACCEPT, Appeal::STATUS_DECLINE, Appeal::STATUS_EXPIRE, Appeal::STATUS_VERIFY, Appeal::STATUS_NOTFOUND, Appeal::STATUS_INVALID, Appeal::STATUS_CHECKUSER];
+
+        $appeals[$appealtypes['assigned']] = Appeal::whereIn('wiki', $wikis)->where(function ($query) use ($basicStatuses) {
+            $query->whereNotIn('status', $basicStatuses)
+            ->where('handlingadmin',Auth::id());
+        })->orWhere(function ($query) use ($isCUAnyWiki) {
+            if ($isCUAnyWiki) {
+                $query->where('status',Appeal::STATUS_CHECKUSER);
+            }
+            else {
+                $query->all();   
+            }
```

Comment:
Not needed. see also https://laravel.com/docs/7.x/queries#conditional-clauses

Reply:
Just removed this as it errors otherwise.